### DISCUSSION TITLE
Widgets only with Astroid Pro-Version

### DISCUSSION THREAD

**Discussion by WM-Loose** [templaza/astroid-framework#909] [General]
Hello Sonny,

there is now also the Astroid Pro version. Can I assume that the new module for integrating the widgets only works if you buy the Astroid Pro version? You once suggested that Astroid has to be profitable. Using the widgets only with the Astroid Pro version would certainly be the right step to generate income. Hence my question as to whether you have thought about it and will offer it in Pro in the future.

Kind regards
Dirk

**Comment by Banuk1616** [comment]
I completely agree with Dirk, if I have understood it correctly. The widget integration module is such a great and unique feature that it should definitely only be available in the Pro version.
Best Regards
